tests/test_search.py

```diff
@@ -8,17 +8,32 @@
 
 import numpy as np
 import pytest
+from scipy.stats import randint, uniform
 
 from surprise import Dataset
 from surprise import Reader
 from surprise import SVD
 from surprise.model_selection import KFold
 from surprise.model_selection import PredefinedKFold
-from surprise.model_selection import GridSearchCV
+from surprise.model_selection import GridSearchCV, RandomizedSearchCV
 from surprise.model_selection import cross_validate
 
 
-def test_parameter_combinations():
+# Tests for GridSearchCV class
+def test_basesearchcv_parse_options():
```

Comment:
Please add short docstring explening the purpose of the test

Reply:
All set!